### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by duramato**
Congrats to the devs 4.0.56 was one of the versions with the most changes

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+* Change: popularShows page to use cached imports
+* Removed: inc_top and inc_bottom mako templates
+
+### 4.0.56 (2015-09-08)
+
+[full changelog](https://github.com/SiCKRAGETV/SickRage/compare/v4.0.55...v4.0.56)
+
+* Added: nzbToMedia scripts in a git submodule, under the contrib folder
+* Added: Total show size column
+* Added: Developer info to footer
```
Comment (MGaetan89) [inline CHANGES.md]: "Debug info"?

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+* Added: Method for auto-fixing shows with tvrage ids to tvdb ids
+* Added: Missing network logos
+* Added: Missing spotweb.be provider icons
+* Added: Show "Add Anime" at the bottom of the anime tables
```
Comment (MGaetan89) [inline CHANGES.md]: This got included in 4.0.58 ;)

Comment (duramato) [inline CHANGES.md]: i dont believe so take a look https://github.com/SiCKRAGETV/SickRage/compare/v4.0.57...v4.0.58
it isnt there

Comment (MGaetan89) [inline CHANGES.md]: It's the first item in the list ;)

Comment (duramato) [inline CHANGES.md]: done if you see anything else go ahead and point :)

**Comment by MGaetan89** [comment]
Looks good to me now :)

**Pull request merged**